client/config/jest/cssTransform.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
```

Comment:
I've been meaning to ask about this syntax for a while; is it not possible to use our usual `export` syntax with jest?

Reply:
![image](https://user-images.githubusercontent.com/14998054/107883326-59892800-6ebc-11eb-8cf1-73cce9577e82.png)


Just tried, and no it doesn't work. I believe this is because jest is reading that without any build step. `export` I believe is ES6 or some newer version of ecmascript that jest doesn't understand or something. The official examples also use `module.exports` syntax so I think it's ok.